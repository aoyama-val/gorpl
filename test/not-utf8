file encoding: CP932
В±ВсВ…ВњВЌРҐКE
hoge
