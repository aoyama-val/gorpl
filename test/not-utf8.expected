file encoding: CP932
В±ВсВ…ВњВЌРҐКE
fuga
